src/3ds_decoder.cpp

```diff
@@ -421,11 +426,11 @@ int OpenAudioDecoder(FILE* stream, DecodedMusic* Sound, std::string const& filen
 		#ifdef MIDI_DEBUG
 		samplerate = 11025;
 		Output::Warning("MIDI track detected, lowering samplerate to 11025 Hz.", filename.c_str());
```

Comment:
Nitpicking a bit, but the second argument is useless here, as there is no %s in the string.